models/news.model.js

```diff
@@ -4,8 +4,18 @@ exports.selectAllTopics = () => {
     return db.query('SELECT * FROM topics')
     .then(({rows}) => {
         if (rows.length === 0) {
-            return Promise.reject({ msg: "not found", status: 403 })
+            return Promise.reject({ msg: "not found", status: 404 })
         }
         return rows
     });
-};
\ No newline at end of file
+};
+
+exports.selectAllUsernames = () => {
+    return db.query('SELECT username FROM users')
+    .then(({rows}) => {
+        if (rows.length === 0) {
+            return Promise.reject({ msg: "not found", status: 404 })
```

Comment:
If there's no users in the database then this isn't necessarily an error. If you were expecting a single row to come back then doing a length check is appropriate but as you are selecting all the rows 0 is a perfectly valid result.

I'd take this check out as it never occurs in your tests and may cause problems later on. Similar for topics above.